diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1010923e..0b17700f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,24 +80,24 @@ if(MSVC)
     if(CMAKE_CL_64)
         # Paths for 64-bit windows builds
         set(OPENSSL_PATH "C:/dev/OpenSSL-Win64" CACHE PATH "OpenSSL Path")
-        set(QT5_PATH "C:/dev/Qt/5.12.12/msvc2017_64" CACHE PATH "Qt5 Path")
+        set(QT5_PATH "C:/Qt/5.15.2/msvc2019_64" CACHE PATH "Qt5 Path")
 
         # Choose between SQLCipher or SQLite, depending whether
         # -Dsqlcipher=on is passed on the command line
         if(sqlcipher)
-            set(SQLITE3_PATH "C:/git_repos/SQLCipher-Win64" CACHE PATH "SQLCipher Path")
+            set(SQLITE3_PATH "C:/dev/SQLCipher-Win64" CACHE PATH "SQLCipher Path")
         else()
             set(SQLITE3_PATH "C:/dev/SQLite-Win64" CACHE PATH "SQLite Path")
         endif()
     else()
         # Paths for 32-bit windows builds
         set(OPENSSL_PATH "C:/dev/OpenSSL-Win32" CACHE PATH "OpenSSL Path")
-        set(QT5_PATH "C:/dev/Qt/5.12.12/msvc2017" CACHE PATH "Qt5 Path")
+        set(QT5_PATH "C:/Qt/5.15.2/msvc2019" CACHE PATH "Qt5 Path")
 
         # Choose between SQLCipher or SQLite, depending whether
         # -Dsqlcipher=on is passed on the command line
         if(sqlcipher)
-            set(SQLITE3_PATH "C:/git_repos/SQLCipher-Win32" CACHE PATH "SQLCipher Path")
+            set(SQLITE3_PATH "C:/dev/SQLCipher-Win32" CACHE PATH "SQLCipher Path")
         else()
             set(SQLITE3_PATH "C:/dev/SQLite-Win32" CACHE PATH "SQLite Path")
         endif()
