diff --git a/installer/windows/product.wxs b/installer/windows/product.wxs
index c040591a..da3c2604 100644
--- a/installer/windows/product.wxs
+++ b/installer/windows/product.wxs
@@ -63,7 +63,7 @@
                     <Directory Id="extensions" Name="extensions">
                         <Component><File Source="$(var.SQLitePath)\math.dll" /></Component>
                         <Component><File Source="$(var.SQLitePath)\formats.dll" /></Component>
-                        <Component><File Source="$(var.SQLitePath)\fileio.dll" /></Component>
+                        <!-- <Component><File Source="$(var.SQLitePath)\fileio.dll" /></Component> -->
                     </Directory>
                     <Directory Id="imageformats" Name="imageformats">
                         <Component><File Source="$(var.QtPath)\plugins\imageformats\qgif.dll" /></Component>
@@ -149,9 +149,9 @@
                 <Feature Id="FormatsExtension" Title="Formats" Description="Provide additional field display formats." AllowAdvertise="no">
                     <ComponentRef Id="formats.dll" />
                 </Feature>
-                <Feature Id="FileioExtension" Title="Fileio" Description="Implements SQL functions readfile(), writefile(), and eponymous virtual type 'fsdir'." AllowAdvertise="no">
+                <!-- <Feature Id="FileioExtension" Title="Fileio" Description="Implements SQL functions readfile(), writefile(), and eponymous virtual type 'fsdir'." AllowAdvertise="no">
                     <ComponentRef Id="fileio.dll" />
-                </Feature>
+                </Feature> -->
             </Feature>
 
             <!-- Registry -->
