diff --git a/installer/windows/variables.wxi b/installer/windows/variables.wxi
index fbedf0c3..368888ef 100644
--- a/installer/windows/variables.wxi
+++ b/installer/windows/variables.wxi
@@ -40,8 +40,8 @@
         Visual Studio 2017. The build "ARCH" will be set automatically.
     -->
 
-    <?define VCRedistPath="C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Redist\MSVC\14.16.27012\MergeModules" ?>
-    <?define VCRedistFile="Microsoft_VC141_CRT_$(sys.BUILDARCH).msm" ?>
+    <?define VCRedistPath="C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\VC\Redist\MSVC\14.29.30133\MergeModules" ?>
+    <?define VCRedistFile="Microsoft_VC142_CRT_$(sys.BUILDARCH).msm" ?>
 
     <!--
     =================================================== Build System ===================================================
@@ -49,20 +49,11 @@
     Paths for the various libraries the application needs to compile. Change these to match the build machine.
     -->
 
-    <?if $(sys.BUILDARCH)="x64"?>
-        <?define QtPath="C:\dev\Qt\5.12.12\msvc2017_64" ?>
-        <?define SQLitePath="C:\dev\SQLite-Win64" ?>
-        <?define SQLCipherPath="C:\git_repos\SQLCipher-Win64" ?>
-        <?define OpenSSLPath="C:\dev\OpenSSL-Win64" ?>
-        <?define SQLiteExePath="C:\builds\release-sqlite-win64\Release" ?>
-        <?define SQLCipherExePath="C:\builds\release-sqlcipher-win64\Release" ?>
-    <?elseif $(sys.BUILDARCH)="x86"?>
-        <?define QtPath="C:\dev\Qt\5.12.12\msvc2017" ?>
-        <?define SQLitePath="C:\dev\SQLite-Win32" ?>
-        <?define SQLCipherPath="C:\git_repos\SQLCipher-Win32" ?>
-        <?define OpenSSLPath="C:\dev\OpenSSL-Win32" ?>
-        <?define SQLiteExePath="C:\builds\release-sqlite-win32\Release" ?>
-        <?define SQLCipherExePath="C:\builds\release-sqlcipher-win32\Release" ?>
-    <?endif?>
+    <?define QtPath="$(env.Qt5Path)" ?>
+    <?define SQLitePath="$(env.SQLitePath)" ?>
+    <?define SQLCipherPath="$(env.SQLCipherPath)" ?>
+    <?define OpenSSLPath="$(env.OpenSSLPath)" ?>
+    <?define SQLiteExePath="$(env.ExePath)\release-sqlite\Release" ?>
+    <?define SQLCipherExePath="$(env.ExePath)\release-sqlcipher\Release" ?>
 
 </Include>
